openapi-diff/src/modeler/AutoRest.Swagger.Tests/AutoRest.Swagger.Tests.csproj

```diff
@@ -1,12 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), common/common.proj ))/common/common.proj" />
   <Import Project="$(common)copy-resource-to-output.proj" />
-  <Import Project="$(common)package-fallback.proj" />
   <Import Project="$(common)project-xunittest.proj" />
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
`netstandard2.0`

Reply:
Same as previous comment. since the unit test project is an `exe` we need to keep `netcoreapp2.0`